components/raft_log_engine/src/engine.rs

```diff
@@ -344,6 +345,7 @@ const STORE_IDENT_KEY: &[u8] = &[0x01];
 const PREPARE_BOOTSTRAP_REGION_KEY: &[u8] = &[0x02];
 const REGION_STATE_KEY: &[u8] = &[0x03];
 const APPLY_STATE_KEY: &[u8] = &[0x04];
+const RECOVER_FROM_RAFT_DB_KEY: &[u8] = &[0x05];
```

Comment:
This is confusing. Why use a different key?

Reply:
Just follow the style that all special keys written to raft engine were defined in the `raft_log_engine` crate.

Reply:
We should either move the key in "keys" to engine_rocks, or unify the key in different engines.

Reply:
Those keys can not be simply moved to engine_rocks, for example the`LOCAL_PREFIX` was also used in raftstore. The right way is abstracting all operations related to keys to methods, which needs a lot of extra work and should not be included in this PR.